pkg/resource/factory.go

```diff
@@ -94,10 +95,14 @@ func (rf *Factory) SliceFromBytes(in []byte) ([]*Resource, error) {
 	for len(kunStructs) > 0 {
 		u := kunStructs[0]
 		kunStructs = kunStructs[1:]
-		if u.GetKind() == "List" {
+		if strings.HasSuffix(u.GetKind(), "List") {
 			items := u.Map()["items"]
 			itemsSlice, ok := items.([]interface{})
 			if !ok {
+				if items == nil {
```

Comment:
Does this cover `items` is not present in `u`?

Reply:
I believe it does - should that be an error? The situation I've encountered most with Ship is when a rendered helm template contains `items:` because they didn't handle the "nothing in list" case

Reply:
Can you add a test case for `items` not present. I'm worried either `items := u.Map()["items"]` or `itemsSlice, ok := items.([]interface{})` would fail.

Reply:
Added

Reply:
Thank you. Looks good to me.